File: code_chef_starters_22/elections_in_chef_land.py
t = int(input())
for i in range(t):
    a = int(input())
    b = int(input())
    c = int(input())
    if(a>b and a>c and a>50):
        print("a")
    elif(b>a and b>c and b>50):
        print("b")
    elif(c>a and c>b and c>50):
        print("c")
    else:
        print("nota")
    

File: code_chef_starters_22/elections_in_chefland.cpp
#include <iostream>
using namespace std;
int main(){
    int t;
    cin>>t;
    for(int i=0;i<t;i++){
        int a,b,c;
        cin>>a>>b>>c;
        if(a>b && a>c && a>50){
            cout<<"a"<<endl;
        }else if(b>a && b>c && b>50){
            cout<<"b"<<endl;
        }else if(c>a && c>b && c>50){
            cout<<"c"<<endl;
        }else{
            cout<<"nota"<<endl;
        }
    }
    return 0;
}
